package.json

```
@@ -41,30 +42,20 @@
     "chai": "4.3.4",
     "eslint": "^7.25.0",
     "express": "4.17.1",
+    "jest": "^26.6.3",
     "mocha": "^8.3.2",
```

Your PR is marked as closing the #77. `mocha` is still necessary? If I understand correctly, you replace it with `jest`
```suggestion
    "jest": "^26.6.3",
```

---

correct, fixing

---

This was probably a merge error

Removed (I think again) `@babel/core`, `@babel/preset-env`, `babelify`, `browserify`, `chai`, `mocha`, `sinon`, `uglyify`